package.json

```diff
@@ -57,7 +57,7 @@
     "@types/jest": "^24.0.13",
     "@types/node": "^13.1.0",
     "@types/react-dom": "^16.8.4",
-    "@types/react-native": "^0.63.0",
+    "@types/react-native": "0.71.6",
```

Comment:
I believe we don't need `` anymore, it's now shipped with the react-native package: https://reactnative.dev/blog/2023/01/12/version-071#typescript-by-default

> We’re also offering built-in, more accurate TypeScript declarations directly from the react-native package. This means you won’t need @types/react-native any longer, and the types will be updated in lockstep with React Native releases.

Reply:
Done 👍

Reply:
Thanks!! I'll merge it and publish the new version